templates/default/jaas_client_config.erb

```diff
@@ -0,0 +1,23 @@
+// Auto-generated by Chef, do not edit!
+
+// JAAS configuration for Kafka client
+KafkaClient {
+    com.sun.security.auth.module.Krb5LoginModule required
+<% node["kafka"]["kerberos"]["krb5_properties"].each do |key, value| -%>
+    <%= key %>=<%= (value.to_s == 'true' || value.to_s == 'false') ? value : "\"#{value}\"" %>
```

Comment:
can you explain this logic? seems like if its true the value isn't quoted if its false it is? is that right?